main.tf

```diff
@@ -2,14 +2,14 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "3.26.0"
+      version = "4.52.0"
     }
     random = {
       source  = "hashicorp/random"
-      version = "3.0.1"
+      version = "3.4.3"
     }
   }
-  required_version = ">= 1.1.0"
+  required_version = "~> 1.3.0"
```

Comment:
This means that it won't work for eg Terraform 1.4.x - is this what we want, or `"~> 1.3"`?

Reply:
I'm not sure what we really want ... maybe I don't need to mess with the version at all. I keep going back and forth on whether and how to constrain TF versions.

But you're probably right that "~> 1.3" makes more sense than "~> 1.3.0" since it should provide just enough protection (against, for instance, breaking changes in 2.x), but should also work for nearly all users who already have terraform installed.

Reply:
I'm just going to leave the version as-is. It runs within GitHub Actions, and the `setup-terraform` GH Action will install the latest version. I'd rather not fiddle too many knobs right now.